extract_token_from_browser.py

```diff
@@ -147,13 +150,35 @@ def get_cookies(cookies_path: StrPath, cookie_query: str) -> tuple[str, str | No
             sys.exit(1)
 
     cookies_path = default_profile_path.joinpath("cookies.sqlite")
+
+    if args.container:
+        try:
+            ctx_id = int(args.container)
+        except ValueError:
+            # non-numeric container ID, try to find by name
+            ctx_id = None
+            with open(default_profile_path.joinpath("containers.json"), "rb") as fp:
+                containers = json.load(fp)
+                for i in containers["identities"]:
+                    if "name" in i and i["name"] == args.container:
+                        ctx_id = i['userContextId']
+                        break
+            if ctx_id is None:
+                print(f"Couldn't find Firefox container '{args.container}'",
+                      file=sys.stderr)
+                sys.exit(1)
+
+        userctx = f"^userContextId={ctx_id}"
+    else:
+        userctx = ""
+
     cookie_query = (
-        "SELECT value FROM moz_cookies WHERE host = '.slack.com' " "AND name = '{}'"
+        f"SELECT value FROM moz_cookies WHERE originAttributes='{userctx}' AND host = '.slack.com' " "AND name = '{}'"
```

Comment:
Since this is just invoked manually by a user on their local machine, I guess SQL injections aren't a concern, but I would prefer this to be an SQL parameter instead of including it directly in the string nevertheless.

Reply:
Sure, can do!